common/changes/@rushstack/set-webpack-public-path-plugin/main_2024-01-11-21-47.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/set-webpack-public-path-plugin",
+      "comment": "Remove old options, specifically `systemJs`, `urlPrefix`, `publicPath`, and `skipDetection`.",
+      "type": "major"
```

Comment:
Unsure if this is an issue, just want to make sure that we don't double-major bump